aws/scripts/py/origin_change.py

```
@@ -25,9 +25,11 @@
     return distribution_ids
 
 
-def fetch_distributions_configs(distribution_ids):
-    print("Fetching distribution configurations...")
-    distribution_configs = []
+def fetch_and_filter_distributions(distribution_ids):
```

Ruff (0.11.9)

Missing return type annotation for public function `fetch_and_filter_distributions`

(ANN201)